lib/presentation/screens/auth/login/login_screen.dart

```diff
@@ -19,12 +19,79 @@ class _LoginScreenState extends State<LoginScreen> {
         listener: (context, state) => _onStateChanged(state),
       );
 
-  // TODO: Implement content
+  final ButtonStyle createAccountButtonStyles = ElevatedButton.styleFrom(
+    primary: Colors.white,
+    onPrimary: Colors.black,
+    minimumSize: const Size(500, 50),
+    shape: const RoundedRectangleBorder(
+        borderRadius: BorderRadius.all(Radius.circular(50)),
+        side: BorderSide(width: 1, color: Colors.white)),
+  );
+
+  final ButtonStyle signInButtonStyles = OutlinedButton.styleFrom(
+    primary: Colors.white,
+    minimumSize: const Size(500, 50),
+    side: const BorderSide(
+      width: 1.0,
+      color: Colors.white,
+    ),
+    shape: const RoundedRectangleBorder(
+      borderRadius: BorderRadius.all(Radius.circular(50)),
+    ),
+  );
+
```

Comment:
So let's begin with that:
we should stick with the pattern of cascade calls, method calls should be as close as possible to the definition by that I mean:

void someMethod() {
  methodA();
  methodC();
 }

void methodA() {
 //sth
 methodB();
}

void methodB() {
//sth
}

void methodC() {
 //sth
}

So transferring it to our case build calls _body and _onStateChange so first we should have all methods for body (it's whole cascade) then _onStateChange